symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkConfig.java

```diff
@@ -25,6 +25,7 @@
 
   private BdkRetryConfig retry = new BdkRetryConfig();
   private BdkDatafeedConfig datafeed = new BdkDatafeedConfig();
+  private BdkCommonJwtConfig commonJwt = new BdkCommonJwtConfig();
```

Comment:
the property will be 
```
commonJwt:
  enabled: true
```
Not sure about the name though.

Reply:
or if we want to make it simpler, just enableCommonJwt (this will be a temporary and hidden feature flag)

Reply:
It is unclear for me if we will really use them (ideally we should), but how do we know in advance the services that are going to be used?
do we put the burden on the bot's developer to decide in advance if he is going to use this API or not?

Reply:
so keeping commonJwt:  (and sub keys) leaves the room for enabled and scopes